.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Self checklist**
+
+ - [ ] I checked that this issue hasn't been created before
+ - [ ] I added any relevant label to the issue
+
+# Describe the solution you'd like
```

Comment:
This is weird title to see in a feature request. 

```suggestion
# Description
```

Reply:
Changed it in https://github.com/inmanta/inmanta/pull/2416/commits/5714de4776cf72764d9697480f3502ed2ae70800